docs/api/wrappers/pz_wrappers.md

```diff
@@ -63,12 +63,13 @@ We wanted our pettingzoo environments to be both easy to use and easy to impleme
 You can apply these wrappers to your environment in a similar manner to the below examples:
 
 To wrap a Parallel environment.
-```python
+```python notest
 from pettingzoo.utils import CaptureStdoutWrapper
 from pettingzoo.butterfly import pistonball_v6
 parallel_env = pistonball_v6.env()
 parallel_env = CaptureStdoutWrapper(parallel_env)
 
+#TODO parallel_env.reset() returns None here
```

Comment:
Oh this isn’t ideal I wonder what caused it. I guess one of the wrappers.

Reply:
Should I check this out too?

Reply:
If you could yeah, parallel envs should return obs, info on reset

Reply:
Oh duh it’s defined as .env instead of .parallel_env — the former calls a conversion script to make it aec. What really should be done as well if the parallel and regular api tests need to assert isinstance(parallelenv) etc